Directory.Packages.props

```diff
@@ -1,12 +1,16 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-  </PropertyGroup>
-
-  <PropertyGroup>
     <OTelLatestStableVer>1.6.0</OTelLatestStableVer>
   </PropertyGroup>
 
+  <Target Name="PreparePackageValidation" BeforeTargets="Restore">
+    <PropertyGroup>
+      <EnablePackageValidation Condition="'$(EnablePackageValidation)' == ''">true</EnablePackageValidation>
```

Comment:
Now that this property defaults to true, you can remove it from the YML here: https://github.com/open-telemetry/opentelemetry-dotnet/blob/d76542e9daa40051a9e5408bc7a849a6c3974684/.github/workflows/package-validation.yml#L22

Reply:
I actually intended this to be `false`! Must have set it to `true` when I switched to the `Restore` phase. Woops! Fixed on #4927